devtools/dev-requirements.txt

```diff
@@ -1,6 +1,10 @@
 # standard install requirements
 -r ../requirements.txt
 
+# use notebooks on cluster, default env won't show-up on
```

Comment:
what do you mean by this line? that you need to call `ipykernel`  with the conda env if you want to use an environment with the jupyter?

this is not cluster specific btw, even on a local laptop, jupyter (and VS code jupyter) (in my experience) do not automatically detect my conda env and so I must install and use ipykernel to register it as an available kernel

Reply:
Yeah it is for jupyter. I was testing on 2 different  laptops simultenously and wrote some dummy comment. I will rephrase it. 

But yeah the idea to include this is, whenever you want to use your new environment, jupyter complains, for vs code it is straightforward but for clusters it is more of an issue to understand why it is not showing up. Either you need to select it before you create interactive session (show all environment even thought they dknt have ipykernel) or install it.